creator-node/src/routes/files.js

```diff
@@ -172,208 +167,106 @@ const getCID = async (req, res) => {
   const CID = req.params.CID
   const trackId = parseInt(req.query.trackId)
 
-  const decisionTree = [{ stage: `BEGIN`, time: `${Date.now()}` }]
-  const logPrefix = `[getCID] [CID=${CID}]`
+  const prometheusResult = { result: null, mode: 'default' }
 
   /**
    * Check if CID is servable from BlacklistManager; return error if not
    */
-  let startMs = Date.now()
   const BlacklistManager = req.app.get('blacklistManager')
   const isServable = await BlacklistManager.isServable(CID, trackId)
-  decisionTree.push({
-    stage: `BLACKLIST_MANAGER_CHECK_IS_SERVABLE`,
-    time: `${Date.now() - startMs}ms`
-  })
   if (!isServable) {
-    decisionTree.push({
-      stage: `CID_IS_BLACKLISTED`
-    })
-
-    logGetCIDDecisionTree(decisionTree, req)
-    return sendResponse(
+    prometheusResult.result = 'abort_delisted'
+    return sendResponseWithMetric(
       req,
       res,
-      errorResponseForbidden(
-        `${logPrefix} CID has been blacklisted by this node`
-      )
+      errorResponseForbidden('CID has been delisted by this node'),
+      prometheusResult
     )
   }
 
+  const storagePaths = []
```

Comment:
k i actually really like the restructuring you did, but would be very helpful to annotate that in PR bc it took me a while to figure out what was happening :)

Reply:
the current comments still explain the what the code does so i don't think this flow needs additional comments

Reply:
no haha i didn't mean in the code, i meant literally in the GH PR diff to make reviewing easier
code is totally fine 🙂